lib/atscppapi/examples/websocket/WebSocket.cc

```diff
@@ -0,0 +1,311 @@
+#include "WebSocket.hh"
+
+#include <iostream>
+#include "ts/ink_base64.h"
+#include "openssl/evp.h"
+#include <netinet/in.h>
+#include <arpa/inet.h>
+
+// DISCLAIMER: this is intended for demonstration purposes only and
+// does not pretend to implement a complete (or useful) server.
+
+using namespace atscppapi;
+
+#define SAY(a) std::cout << a << std::endl;
```

Comment:
Please see the stuff we have available in atscppapi/Logger.h, such as TSDebug or creating a logger class.